lib/wizard/builder.rb

```diff
@@ -37,108 +47,33 @@ def build
         end
       end
 
-      @wizard.append_step('forum-title') do |step|
-        step.add_field(id: 'title', type: 'text', required: true, value: SiteSetting.title)
-        step.add_field(id: 'site_description', type: 'text', required: false, value: SiteSetting.site_description)
-        step.add_field(id: 'short_site_description', type: 'text', required: false, value: SiteSetting.short_site_description)
-
-        step.on_update do |updater|
-          updater.ensure_changed(:title)
-
-          if updater.errors.blank?
-            updater.apply_settings(:title, :site_description, :short_site_description)
-          end
-        end
-      end
-
-      @wizard.append_step('introduction') do |step|
-        introduction = IntroductionUpdater.new(@wizard.user)
```

Comment:
Do we need to delete `IntroductionUpdater` and its spec file? As far as I can see this is the only spot where it's used.

Reply:
I was planning to delete those files in follow-up PR, but I deleted them now here.